### PR TITLE
Removes several printStackTrace dumped into stderr

### PR DESCRIPTION
**Pull request by fredoboulo**
This PR does not remove all of them, and I still get some doubts about some parts. I will comment the doubtful lines in the PR itself.

### CODE REVIEW

File: src/main/java/org/zeromq/ZStar.java [hunk 1]
```diff
@@ -520,7 +520,6 @@ public void run(final Object[] train, final ZContext chef, final Socket mic)
                         }
                         catch (Exception e) {
                             // really ?
-                            e.printStackTrace();
```
Comment (daveyarwood) [inline src/main/java/org/zeromq/ZStar.java]: I'm curious about this one. Swallowing exceptions is usually bad news, especially unspecific exceptions. Probably better to remove the try/catch altogether and let exceptions happen, if this is a "this should never happen" scenario. Or we could wrap it in some kind of ZMQ exception wrapper class.

**Comment by fredoboulo** [comment]
@daveyarwood If you are about to make a release, merging this PR can wait. Last-second changes? Mmmm better wait actually. I will close the PR and reopen it later.

**Comment by daveyarwood** [comment]
@fredoboulo Sounds good :)  

I think @trevorbernard will be cutting a release soon.

**Comment by trevorbernard** [comment]
Yeah, 0.4.3 is almost 1 year old.

**Pull request closed without merging**